.gitlab/source_test/macos.yml

```diff
@@ -117,7 +137,13 @@ lint_macos_gitlab_arm64:
 tests_macos_gitlab_amd64:
   extends: .tests_macos_gitlab
   tags: ["macos:monterey-amd64", "specific:true"]
+  after_script:
+    - !reference [.upload_junit_source]
+    - !reference [.upload_coverage]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Would it be better to add another reference that combines both of them? It seems that they are always used together 🤷 (Something like `.upload_reports`?)

Reply:
We will combine them in the end, but for now it's still not used together everywhere (for arm jobs for example)